config/config.go

```diff
@@ -34,6 +34,7 @@ type Config struct {
 	EthTxManager ethtxmanager.Config `mapstructure:"EthTxManager"`
 	L1           L1Config            `mapstructure:"L1"`
 	Telemetry    Telemetry           `mapstructure:"Telemetry"`
+	KMSKeyName   string
```

Comment:
How this config value gets populated? Do we need `mapstructure` for it as well if it is populated from the `aggalayer.toml`?

Reply:
Moved to ethtxmanager config